_posts/2020-05-15-issue-2.md

```diff
@@ -0,0 +1,71 @@
+
+[California is suing Uber and Lyft](https://www.nytimes.com/2020/05/05/technology/california-uber-lyft-lawsuit.html) for misclassifying drivers as independent contractors rather than employees under [AB-5](https://en.wikipedia.org/wiki/California_Assembly_Bill_5_(2019)), a new state law that took effect on January 1, 2020. This is a significant win for workers and organizing groups like [Rideshare Drivers United](https://drivers-united.org) who have been advocating for its implementation and enforcement. Rideshare drivers are tech workers, too. If you are a driver, [you can file a wage claim here](https://drivers-united.org/a/submit-a-wage-claim). 
+
+In Chicago, Instacart shopper and organizer Matthew Telles of [Gig Workers Collective](https://www.gigworkerscollective.org) gives a personal look into the deteriorating working conditions that prompted gig workers to fight back in [an interview for Picture Theory](https://podcasts.apple.com/us/podcast/matthew-telles-gig-workers-are-standing-up-to-big-tech/id1361867735?i=1000473489342). 
+
+[Hate groups promoting violence](https://www.metrotimes.com/news-hits/archives/2020/05/11/whitmer-becomes-target-of-dozens-of-threats-on-private-facebook-groups-ahead-of-armed-rally-in-lansing) are continuing to flourish in private Facebook groups. *“Dozens of angry Michiganders, fueled by conspiracy theories and disinformation about the coronavirus, are promoting violence and mobilizing armed rallies against Gov. Gretchen Whitmer on Facebook, in violation of the social media company’s policies.”* Facebook’s inaction and complacency are unacceptable during this unprecedented public health crisis. Meanwhile, 11,250 current and former content moderators working for Facebook globally &mdash; from [Phoenix](https://www.theverge.com/2019/2/25/18229714/cognizant-facebook-content-moderator-interviews-trauma-working-conditions-arizona) to [the Philippines](https://www.vice.com/en_us/article/ywe7gb/the-companies-cleaning-the-deepest-darkest-parts-of-social-media) &mdash; were awarded [a $52M settlement](https://www.theverge.com/2020/5/12/21255870/facebook-content-moderator-settlement-scola-ptsd-mental-health) of at least $1,000 each. This is a major symbolic victory but a minor cost to Facebook, and a very small amount to compensate for what may be life-long post-traumatic stress for handling some of the worst content on the web.
```

Comment:
Not sure about the intent of the part that's bolded (that starts "Dozens of angry Michiganders..."). Is it a quote from the linked article? Quotes in other sections seem to be emphasized with italics, using `_` instead of `*`, so that seems inconsistent.

Reply:
Ah yes, single `_` and `*` are italics, double are bold.

Reply:
i'll switch to use `_`